pkg/client/unversioned/request.go

```diff
@@ -238,6 +246,35 @@ func (r *Request) AbsPath(segments ...string) *Request {
 	} else {
 		r.path = path.Join(segments...)
 	}
+	r.setAbspath = true
+	return r
+}
+
+// Group
+func (r *Request) Group(group string) *Request {
```

Comment:
If people have group & version as separate strings, we don't want to make
them concatenate.

On Fri, Sep 18, 2015 at 11:15 AM, Chao Xu <notifications@github.com> wrote:

> In pkg/client/unversioned/request.go
> <https://github.com/kubernetes/kubernetes/pull/14149#discussion_r39884634>
> :
>
> > @@ -238,6 +246,35 @@ func (r *Request) AbsPath(segments ...string) *Request {
> >  	} else {
> >  		r.path = path.Join(segments...)
> >  	}
> > +	r.setAbspath = true
> > +	return r
> > +}
> > +
> > +// Group
> > +func (r *Request) Group(group string) *Request {
>
> Could you provide a use case of Group? When would someone set the group
> but not the version at the same time?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14149/files#r39884634>.
>